src/main/java/net/minecraftforge/fml/common/registry/VillagerRegistry.java

```diff
@@ -160,7 +162,6 @@ public void registerVillageCreationHandler(IVillageCreationHandler handler)
      * @param defaultSkin
      */
     @SideOnly(Side.CLIENT)
-    @Deprecated // Doesn't work at all.
```

Comment:
The profession should have a texture location in it already so this function shouldn't be necessary at all.